src/components/GoodList.tsx

```diff
@@ -0,0 +1,15 @@
+import { Good } from '../types/Good.type';
+
+type Props = {
+  goods: Good[];
+};
+
+export const GoodList: React.FC<Props> = ({ goods }) => (
+  <ul>
+    {goods.map(good => (
+      <li key={good} data-cy="Good">
+        {good}
```

Comment:
[CODE KNOWLEDGE] - Directly rendering `good` suggests that it is a string or a number. If `good` is an object, you should not render it directly to the DOM. Instead, render its properties or create a component to handle the display of `good`.

Reply:
good is a string